### ISSUE TITLE
Installation instructions for @humanmade/eslint-config are incorrect

### ISSUE THREAD

**Issue by johnbillion** [humanmade/coding-standards#221]
I'm trying to set up a fresh project which uses `@humanmade/eslint-config` for its linting. The installation instructions at https://github.com/humanmade/coding-standards/blob/master/packages/eslint-config-humanmade/readme.md appear to be incorrect as I only get errors after following them.

## Steps to reproduce:

1. Create a new JS project with no existing dependencies
2. Add some JS files to a `src` directory
3. Follow the setup instructions at https://github.com/humanmade/coding-standards/blob/master/packages/eslint-config-humanmade/readme.md
4. Add a `.eslintrc.json` file containing the following:  

       {
         "extends": "humanmade",
       }

4. Run `./node_modules/.bin/eslint -c humanmade src`

Observe the following error message:

```
Error: Cannot read config file: /Users/john/projects/projectname/humanmade
```

I found #176 which looks like the `humanmade` package is now `@humanmade/eslint-config` so I changed the `extends` property to that and now I get the following error:

```
Error: Cannot find module 'eslint-plugin-jsdoc'
```

Am I doing something wrong or are the installation instructions incorrect?

**Comment by ntwb** [comment]
Ooops, didn't mean to close this

**Comment by ntwb** [comment]
### Setup

Starting with a base `package.json` file:
```json
{
  "name": "221-coding-standards",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {},
  "devDependencies": {
    "@humanmade/eslint-config": "^1.0.0"
  }
}

```

And an `.eslintrc.json` file:
```json
{
	"extends": "@humanmade/eslint-config"
}

```

### Install

Then running:

`npx install-peerdeps --dev @humanmade/eslint-config@latest`

### Actual Result

```shell
❯ npx install-peerdeps --dev @humanmade/eslint-config@latest
install-peerdeps v2.0.3
Installing peerdeps for @humanmade/eslint-config@latest.
npm install @humanmade/eslint-config@1.0.0 babel-eslint@^10.0.0 eslint@^5.10.0 eslint-config-react-app@^3.0.5 eslint-plugin-flowtype@^3.2.0 eslint-plugin-import@^2.14.0 eslint-plugin-jsx-a11y@^6.1.2 eslint-plugin-react@^7.11.1 eslint-plugin-react-hooks@^4.0.2 --save-dev --registry https://registry.npmjs.org

npm WARN eslint-config-react-app@3.0.8 requires a peer of babel-eslint@9.x but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-react-app@3.0.8 requires a peer of eslint-plugin-flowtype@2.x but none is installed. You must install peer dependencies yourself.
npm WARN 221-coding-standards@1.0.0 No description
npm WARN 221-coding-standards@1.0.0 No repository field.

+ eslint-plugin-flowtype@3.13.0
+ babel-eslint@10.1.0
+ eslint-plugin-react-hooks@4.1.0
+ eslint-plugin-react@7.20.6
+ eslint-config-react-app@3.0.8
+ eslint-plugin-jsx-a11y@6.3.1
+ eslint-plugin-import@2.22.0
+ @humanmade/eslint-config@1.0.0
+ eslint@5.16.0
added 224 packages from 167 contributors, updated 1 package and audited 225 packages in 9.404s

24 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

SUCCESS @humanmade/eslint-config and its peerDeps were installed successfully.
```

Testing the CLI:
```shell
❯ ./node_modules/.bin/eslint -c @humanmade/eslint-config src
```

Results in `Error: Cannot find module 'eslint-plugin-jsdoc'`

### Expected Result


----

The `peerDependencies` should install all required `devDependencies`, alas it does not currently

Via https://unpkg.com/@humanmade/eslint-config@1.0.0/package.json
```json
"peerDependencies": {
	"babel-eslint": "^10.0.0",
	"eslint": "^5.10.0",
	"eslint-config-react-app": "^3.0.5",
	"eslint-plugin-flowtype": "^3.2.0",
	"eslint-plugin-import": "^2.14.0",
	"eslint-plugin-jsx-a11y": "^6.1.2",
	"eslint-plugin-react": "^7.11.1",
	"eslint-plugin-react-hooks": "^4.0.2"
},
```

This should be installing all of the ESLint configs as `devDependencies` as this package does not distribute these packages:
- `babel-eslint`
- `eslint`
- `eslint-config-react-app`
- `eslint-plugin-flowtype`
- `eslint-plugin-import`
- `eslint-plugin-jsdoc`
- `eslint-plugin-jsx-a11y`
- `eslint-plugin-react`
- `eslint-plugin-react-hooks`
- `eslint-plugin-sort-destructure-keys`

It is not currently installing:

- `eslint-plugin-jsdoc`
- `eslint-plugin-sort-destructure-keys`

The above 2 ESLint shared configs not being included was reported in #215 and fixed in #218, though #218 has not yet been deployed

### Workaround:

The following should work as a workaround until #215/#218 has been published

```shell
npm i eslint-plugin-sort-destructure-keys eslint-plugin-jsdoc --save-dev
```

Results:

```shell
❯ ./node_modules/.bin/eslint -c @humanmade/eslint-config src
Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.

/Users/netweb/Code/HumanMade/tmp/221-coding-standards/src/Gruntfile.js
  3:18  error  Missing JSDoc comment  jsdoc/require-jsdoc

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

----

I've also updated the Readme in #223 to address the `humanmade` references to `@humanmade/eslint-config`

**Comment by kadamwhite** [comment]
The PRs referenced above should have been merged and released by now, so I'm going to close this -- thank you @ntwb for working through the issue here!
